docs/api.md

```diff
@@ -1273,8 +1273,8 @@ or updates existing device details for this user/session
 If no device `id` is specified,
 both `name` and `type` must be provided.
 If a device `id` is specified,
-at least one of `name`, `type`, `pushCallback` and `pushPublicKey`
-must be present.
+at least one of `name`, `type`, `pushCallback` or the couple `pushPublicKey` and `pushAuthKey`
+must be present. Be careful, `pushPublicKey` and `pushAuthKey` can change if `pushCallback` changes
```

Comment:
I took @jrconlin's comment to mean, either we should have *all three* of the push-related fields changing, or none of them.  @jrconlin is that accurate?  Is there ever a situation in which that would not be the case?